pandas/core/indexes/datetimes.py

```diff
@@ -702,23 +702,27 @@ def inferred_type(self) -> str:
 
     def indexer_at_time(self, time, asof=False):
         """
-        Return index locations of index values at particular time of day
-        (e.g. 9:30AM).
+        Return the index location at certain times of the day (eg 9:30 am).
 
         Parameters
         ----------
         time : datetime.time or str
-            datetime.time or string in appropriate format ("%H:%M", "%H%M",
+            A datetime.time or string in appropriate format ("%H:%M", "%H%M",
             "%I:%M%p", "%I%M%p", "%H:%M:%S", "%H%M%S", "%I:%M:%S%p",
             "%I%M%S%p").
+        asof : bool
 
         Returns
         -------
-        values_at_time : array of integers
+        array of integers
+            An array is a container object that holds a fixed number of 
```

Comment:
This line has a trailing space. Should be removed.
```suggestion
            An array is a container object that holds a fixed number of
```